System/EnemyArchetypes.ini

```diff
@@ -2195,7 +2195,7 @@ PrimaryWeapon=(Weapon="SwatEquipment.M1Super90SG",Chance=10)
 PrimaryWeapon=(Weapon="SwatEquipment.Beretta92FSHG",Chance=10)
 PrimaryWeapon=(Weapon="SwatEquipment.BadguyColtM1911HG",Chance=5)
 PrimaryWeapon=(Weapon="",Chance=5)
```

Comment:
what about this? here is no weapon at all

Reply:
5% chance for no secondary.